package.json

```diff
@@ -58,22 +57,21 @@
     "styled-components": "^5.2.3",
     "swr": "^1.1.1",
     "utf-8-validate": "^5.0.7",
-    "uuid": "^8.3.2",
+    "uuid": "9.0.0-beta.0",
```

Comment:
this was required to work with the next.js swc compiler. esm stuffs

Reply:
lul